src/components/ApplicationRoutes/ApplicationRoutes.tsx

```diff
@@ -26,23 +26,50 @@ const NotFoundView = () => (
 export const AUTH_ERROR_MESSAGE =
   'Что-то пошло не так. Для повторного входа в\u00A0систему введите свои e-mail и\u00A0пароль';
 
+const useForceUpdate = () => {
+  const [, set] = useState(true);
+  return useCallback(() => set((s) => !s), [set]);
```

Comment:
еще можно через `useReducer`, тогда `useCallback` не нужен
```suggestion
  const [, forceUpdate] = useReducer((s) => s + 1, 0);
  return forceUpdate;
```

Reply:
исправил